src/pybind/mgr/volumes/fs/operations/subvolume.py

```diff
@@ -59,6 +59,12 @@ def remove_subvol(mgr, fs, vol_spec, group, subvolname, force=False, retainsnaps
         subvolume.remove(retainsnaps)
 
 
+def get_subvol_stats(mgr, fs, volspec, group, subvolname):
+    op_type = SubvolumeOpType.REMOVE
```

Comment:
Why use optype REMOVE ?

Reply:
ping?

Reply:
This piece of code has been deleted. Marking this thread resolved.